new-structure-proposals.md

```diff
@@ -191,9 +191,59 @@ Observations:
 
 5. [Camp 5: Cairo](./camp_5/) - Ben
    1. C(pu)AIR(o)
-   2. Computational Integrity
-   3. Syntax
-   4. Cairo VM
+      - Polynomial Equations
```

Comment:
Are polynomial equations **really** required to understand the Cairo VM, would they not be more useful when understanding the STARKs in Camp 5?

Reply:
As it's part of the definition:

We describe a single set of polynomial equations for
the statement that the execution of a program on this architecture is
valid.

I would say a high level description is warranted. What do you think?

Reply:
agree with Ben